modules/util/source-formatter/src/test/java/com/liferay/source/formatter/processor/UpgradeCatchAllCheckTest.java

```diff
@@ -168,7 +172,15 @@ private static boolean _hasValidUpgradeFiles(
 			"src/test/resources/com/liferay/source/formatter/dependencies" +
 				"/upgrade/upgrade-catch-all-check/" + fileName);
 
-		return Files.exists(filePath);
+		Path expectedFilePath = Paths.get(
+			"src/test/resources/com/liferay/source/formatter/dependencies" +
+				"/expected/upgrade/upgrade-catch-all-check/" + fileName);
+
+		if (Files.exists(filePath) && Files.exists(expectedFilePath)) {
+			return true;
+		}
+
+		return false;
```

Comment:
@drewbrokke Is it fine to create another variable name here? I figured it shouldnt be inlined, but I just want to double check. Also this is how SF wanted me to do the return instead of 
`return Files.exists(filePath) && Files.exists(expectedFilePath);` Does this section look fine to you?

Reply:
Yes, that is ok. You can try to inline it then run SF. There is actually a rule that will instruct you to create a new variable anyways if it is too long. If SF is happy with the current state of things, then we can leave it.

As for the the `return true` / `return false`, that is a very common pattern across portal and works great here.